extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -579,7 +411,7 @@ public function getCuratedContentSection() {
 			throw new NotFoundApiException( 'Section is not set' );
 		}
 
-		$data = $this->getCuratedContentData( $section );
+		$data = MercuryApiMainPageHandler::getCuratedContentData( $this->mercuryApi, $section );
```

Comment:
Above we have `MercuryApiArticleHandler::getArticleData( $article, $this->request, $this->mercuryApi )`
Would be nice to unify how we order params.